git/move_milestones.go

```diff
@@ -0,0 +1,87 @@
+package git
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/go-github/github"
+)
+
+func MoveMilestone(ghClient *github.Client, repo string, oldMilestoneName string, newMilestoneName string, includeClosed bool) error {
+	is := ghClient.Issues
+	milestones, _, err := is.ListMilestones("deis", repo, &github.MilestoneListOptions{})
+	if err != nil {
+		return err
+	}
+	var oldMilestone *github.Milestone
+	for _, milestone := range milestones {
+		if *milestone.Title == oldMilestoneName {
+			oldMilestone = &milestone
+			break
+		}
+	}
+	if oldMilestone == nil {
+		return fmt.Errorf("milestone %s not found", oldMilestoneName)
+	}
+	var newMilestone *github.Milestone
+	for _, milestone := range milestones {
+		if *milestone.Title == newMilestoneName {
+			newMilestone = &milestone
+			break
+		}
+	}
+	if newMilestone == nil {
+		return fmt.Errorf("milestone %s not found", newMilestoneName)
+	}
```

Comment:
I'll do that. I had the same thought, initially, and I'd talked myself out of doing it because I didn't want to get the list of milestones twice. But I _can_ get that list once and just _pass_ it to the same helper function twice. I'm not sure why I didn't. Fixing.